### DISCUSSION TITLE
Battery level

### DISCUSSION THREAD

**Discussion by matt199394** [Edzelf/ESP32Radio-V2#157] [Q&A]
Hi, I have build ESPradio V2 with TFT red display, SD card, DAC, rotary encoder, IR receiver and resistances partition to read battery level by ADC pin. All works well, but I can't visualize the level battery on the screen.
Someone know how to fix the problem.
Thank you

**Comment by Edzelf** [comment]
The battery level indicator is a horizontal line, just below the "ESP32-Radio" text.  The left (green) part has a length that is proportional to the current battery capacity (0...100 percent).  The rest of the line is red.
The line is displayed by the function "bluetft_displaybattery_displaybattery()".
It is important that the ADC values for 0 and 100 percent are set.  See paragraph 2.3.3 of the documentation.

**Comment by matt199394** [comment]
Thank you very much Ed,
I have made a custom PCB, I can share the PCB in the discussion forum if you want
Let me know


Inviato da Outlook per Android<https://aka.ms/AAb9ysg>
________________________________
